src/status_im2/contexts/quo_preview/wallet/account_overview.cljs

```diff
@@ -0,0 +1,92 @@
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "Time frame string"
+    :key   :time-frame-string
+    :type  :text}
+   {:label "Time frame to string"
+    :key   :time-frame-to-string
+    :type  :text}
+   {:label "Percentage change"
+    :key   :percentage-change
+    :type  :text}
+   {:label "Currency change"
+    :key   :currency-change
+    :type  :text}
+   {:label "Current value"
+    :key   :current-value
+    :type  :text}
+   {:label "Loading"
+    :key   :loading?
+    :type  :boolean}
+   {:label   "Metrics"
+    :key     :metrics
+    :type    :select
+    :options [{:key   "Positive"
+               :value :positive}
+              {:key   "Negative"
+               :value :negative}]}
+   {:label "Account name"
+    :key   :account-name
+    :type  :text}
+   {:label   "Account"
+    :key     :account
+    :type    :select
+    :options [{:key   :watched-address
+               :value "Watched address"}
+              {:key   :default
+               :value "Default"}]}
+   {:label   "Time frame"
+    :key     :time-frame
+    :type    :select
+    :options [{:key   :one-week
+               :value "1 Week"}
+              {:key   :one-month
+               :value "1 Month"}
+              {:key   :three-months
+               :value "3 Months"}
+              {:key   :one-year
+               :value "1 Year"}
+              {:key   :all-time
+               :value "All time"}
+              {:key   :custom
+               :value "Custom"}]}])
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:metrics              :positive
+                             :currency-change      "€0.00"
+                             :percentage-change    "0.00%"
+                             :current-value        "€0.00"
+                             :account-name         "Diamond Hand"
+                             :time-frame           :custom
+                             :time-frame-string    "16 May"
+                             :time-frame-to-string "25 May"
+                             :account              :default})]
+    (fn []
+      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+       [rn/view {:padding-bottom 150}
+        [rn/view {:flex 1}
+         [preview/customizer state descriptor]]
+        [rn/view
+         {:padding-vertical 60
+          :flex-direction   :row
+          :justify-content  :center}
+         [quo2/account-overview @state]]]])))
+
+(defn preview-account-overview
+  []
+  [rn/view
+   {:background-color (colors/theme-colors colors/white
+                                           colors/neutral-95)
+    :flex             1}
+   [rn/flat-list
+    {:flex                         1
+     :keyboard-should-persist-taps :always
+     :header                       [cool-preview]
+     :key-fn                       str}]])
```

Comment:
@ibrkhalil  Please add the style properties inside the `:style` key for these namespace too

Reply:
That's the preview namespace, And not the component
I don't think we follow that convention here

Reply:
@ibrkhalil The convention is used in all namespaces, it is not related to production code, it's related to code readability

Reply:
I think it's the same confusion I had before. To be clear, can you put it in a ':style' key and not in a style.cljs file :)

Reply:
Sorry I got confused and thought something else, I totally agree
Doing it now!